### PR TITLE
We give talks about contribex, this lists some good ones in our README

### LINKED ISSUE #5086: Publish list of community reports for SIG ContribEx
We should inline link our community reports and videos right in our README to provide new people a backlog of info

/assign
/sig contributor-experience

### PR DESCRIPTION
**Pull request by castrojo**
Fixes #5086 

Signed-off-by: Jorge O. Castro <jorgec@vmware.com>

### CODE REVIEW

File: sig-contributor-experience/README.md [hunk 1]
```diff
@@ -138,4 +138,29 @@ Creates and maintains tools and automation for Kubernetes Slack.
 [subproject-definition]: https://github.com/kubernetes/community/blob/master/governance.md#subprojects
 <!-- BEGIN CUSTOM CONTENT -->
 
+## Relevant Presentations
+
+This is a collection of presentations from our SIG members around contributing to Kubernetes, feel free to PR in content here: 
+
+## Current Status
```
Comment (mrbobbytables) [inline sig-contributor-experience/README.md:145]:
```suggestion
## Community Updates
```

File: sig-contributor-experience/README.md [hunk 2]
```diff
@@ -138,4 +138,29 @@ Creates and maintains tools and automation for Kubernetes Slack.
...
+
+This is the work we've done this past cycle and plan to work on in the future:
```
Comment (mrbobbytables) [inline sig-contributor-experience/README.md:147]:
```suggestion
```

File: sig-contributor-experience/README.md [hunk 3]
```diff
@@ -138,4 +138,29 @@ Creates and maintains tools and automation for Kubernetes Slack.
...
+
+- [Contributor Experience Community Update](https://youtu.be/J3O8fXTm3HE?t=148)
+  - [Slides](https://docs.google.com/presentation/d/1BS7liUs1WDSPnpNXIHRtRSKhGzK9usS5qzpry_9YU-I/edit?usp=sharing)
```
Comment (mrbobbytables) [inline sig-contributor-experience/README.md:150]:
```suggestion
- July 2020 Community Update - [Recording] (https://youtu.be/J3O8fXTm3HE?t=148) | [Slides](https://docs.google.com/presentation/d/1BS7liUs1WDSPnpNXIHRtRSKhGzK9usS5qzpry_9YU-I/edit?usp=sharing)
```

File: sig-contributor-experience/README.md [hunk 4]
```diff
@@ -138,4 +138,30 @@ Creates and maintains tools and automation for Kubernetes Slack.
...
+- [How to Contribute to Kubernetes](https://www.youtube.com/watch?v=yVOCRrK6Fh0) - Nikhita Raghunath
+- [Practical git for Kubernetes Contributors](https://www.youtube.com/watch?v=K5qI9bTG9Ik)- Christoph Blecker
+- [State of the Kubernetes Contributor Community](https://www.youtube.com/watch?v=a17FLjVDUOc) - Paris Pittman
+- [A peek behind the curtain: Managing the Kubernetes Contributor Community](https://www.youtube.com/watch?v=DrCPEq3HpwY)
+
+### KubeCon + CloudNativeCon
+
+We give our SIG status at every KubeCon, here are our most current talks:
+
+- [Introcution to Contributor Experience 2020](https://docs.google.com/presentation/d/15CaNvfNRDjAaJ1gRmPq2thti5so5m7x5XTCN-NFlPOc/edit?usp=sharing) (Slides only, video will be added once it's published)
```
Comment (cblecker) [inline sig-contributor-experience/README.md:163]:
```suggestion
- [Introduction to Contributor Experience 2020](https://docs.google.com/presentation/d/15CaNvfNRDjAaJ1gRmPq2thti5so5m7x5XTCN-NFlPOc/edit?usp=sharing) (Slides only, video will be added once it's published)
```

File: sig-contributor-experience/README.md [hunk 5]
```diff
@@ -138,4 +138,28 @@ Creates and maintains tools and automation for Kubernetes Slack.
...
 <!-- BEGIN CUSTOM CONTENT -->
 
+## Relevant Presentations
+
+This is a collection of presentations from our SIG members around contributing to Kubernetes, feel free to PR in content here: 
+
+## Community Updates
+
+
+- July 2020 Community Update - [Recording] (https://youtu.be/J3O8fXTm3HE?t=148) | [Slides](https://docs.google.com/presentation/d/1BS7liUs1WDSPnpNXIHRtRSKhGzK9usS5qzpry_9YU-I/edit?usp=sharing)
```
Comment (mrbobbytables) [inline sig-contributor-experience/README.md:148]:
```suggestion
- July 2020 Community Update - [Recording](https://youtu.be/J3O8fXTm3HE?t=148) | [Slides](https://docs.google.com/presentation/d/1BS7liUs1WDSPnpNXIHRtRSKhGzK9usS5qzpry_9YU-I/edit?usp=sharing)
```
caught a space in there XD

Comment (castrojo) [reply sig-contributor-experience/README.md]: My change supercedes this and has the date.

**Pull request merged**